.changeset/slow-ducks-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/babel-plugin': minor
```

Comment:
I think this should be a patch

Please mention the fix in the description, eg "Fixed shadow variables clashing with destructured props"